mirror-service/src/main/java/org/hyades/osv/OsvToCyclonedxParser.java

```diff
@@ -36,156 +36,164 @@
 public class OsvToCyclonedxParser {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(OsvToCyclonedxParser.class);
-    final Bom cyclonedxBom;
-    String ecosystem;
-    Vulnerability.Rating.Severity severity = Vulnerability.Rating.Severity.UNKNOWN;
+    private static final String SEVERITY = "severity";
+    private static final String DATABASE_SPECIFIC = "database_specific";
```

Comment:
May be we don't need these variables for specific two json fields. Either, we create it for each field or we skip all and use them directly while accessing the object.

Reply:
These fields are actually repeated 3 times. I can remove it but more chances of error if same string is repeated

Reply:
Alternatively we could just use Jackson and map the JSON to POJOs instead of parsing it manually. Would reduce code complexity, and field names are only defined once in the respective POJO.

Reply:
Actually that would be the good and right approach. I was not sure about some of the extraction and manipulations like adding strings like 'very:' and concatenating with ecosystem etc